wgpu-hal/src/gles/adapter.rs

```diff
@@ -549,6 +549,13 @@ impl super::Adapter {
             );
         }
 
+        features.set(
+            wgt::Features::FLOAT32_FILTERABLE,
+            extensions.contains("GL_ARB_color_buffer_float")
+                || extensions.contains("GL_EXT_color_buffer_float")
+                || extensions.contains("OES_texture_float_linear"),
+        );
+
```

Comment:
This looks good but we can now remove `PrivateCapabilities::TEXTURE_FLOAT_LINEAR` and just check for the presence of `Features::FLOAT32_FILTERABLE`.